odl/contrib/datasets/ct/examples/mayo_reconstruct.py

```diff
@@ -1,29 +1,38 @@
 """Reconstruct Mayo dataset using FBP and compare to reference recon.
 
 Note that this example requires that Mayo has been previously downloaded and is
-stored in the location indicated by "mayo_dir".
+stored in the location indicated by "proj_folder" and "rec_folder".
 
 In this example we only use a subset of the data for performance reasons,
-there are ~48 000 projections in the full dataset.
+there are ~32 000 projections per patient in the full dataset.
 """
-
+import numpy as np
 import odl
 from odl.contrib.datasets.ct import mayo
+from time import perf_counter
 
-mayo_dir = ''  # replace with your local folder
+# define data folders
+proj_folder = odl.__path__[0] + '/../../data/LDCT-and-Projection-data/' \
+             'L004/08-21-2018-10971/1.000000-Full dose projections-24362/'
+rec_folder = odl.__path__[0] + '/../../data/LDCT-and-Projection-data/' \
+             'L004/08-21-2018-84608/1.000000-Full dose images-59704/'
```

Comment:
- I suggest to use `odl.contrib.datasets.util.get_data_dir()` to determine a storage location.
- What are the reasons to change the dataset directory? It would be good to have a small comment on top that says which data it is.
- Please use `dir` instead of `folder`. The latter is Windows lingo, and Python uses e.g. `is_dir()` to check if a `Path` object represents a directory.

Reply:
The Mayo dataset cannot be downloaded directly from a url but it requires its own software. This prevents to use `odl.contrib.datasets.util.get_data(filename, subset, url)` for local caching. Would this work better?
```suggestion
# replace with your local directory
mayo_dir = ''
# define projection and reconstruction data directories
# e.g. for patient L004 full dose CT scan:
proj_dir = os.path.join(
    mayo_dir, 'L004/08-21-2018-10971/1.000000-Full dose projections-24362/')
rec_dir = os.path.join(
    mayo_dir, 'L004/08-21-2018-84608/1.000000-Full dose images-59704/')
```